app/src/main/java/com/example/campusguide/Constants.kt

```diff
@@ -36,4 +36,7 @@ object Constants {
         45.458153, -73.640490
     )
 
-}
\ No newline at end of file
+    const val PRIMARY_COLOR = "#008577"
+    const val PRIMARY_COLOR_DARK = "#00574B"
+    const val ACCENT_COLOR = "#D81B60"
```

Comment:
These colors are already defined in the colors resource file, maybe we could use those?

Reply:
I forgot you could access them from the XML using `R.color` so I will use the values defined in the colors resource file in that case!

Reply:
Actually never mind, I tested out `R.color` and it crashes so I'm not sure if it's possible. For now I'll add them as constant values in `Constants.kt` and if I find a way to reference the colors from the XML then I'll do that.